Build failure: services.frigate

### Steps To Reproduce

Steps to reproduce the behavior:
1. services.frigate = true;

### Build log

```
error: builder for '/nix/store/rizcrqqa159z3gcb6dblrnbyybzjyw25-nginx-1.26.2.drv' failed with exit code 2;
       last 10 log lines:
       > gcc -c -pipe  -O -W -Wall -Wpointer-arith -Wno-unused-parameter -Werror -g  -DNDK_SET_VAR -DNDK_UPSTREAM_LIST  -I src/core -I src/event -I src/event/modules -I src/event/quic -I src/os/unix -I /nix/store/2ysp5ichpccf4lv1wp2qcwz0bmm840f1-rtmp -I /nix/store/w218m7m6bma3n6hnk15wxw5fd7sn9y7x-develkit/objs -I objs/addon/ndk -I /usr/include/libxml2 -I objs -I src/http -I src/http/modules -I src/http/v2 -I /nix/store/w218m7m6bma3n6hnk15wxw5fd7sn9y7x-develkit/src -I /nix/store/w218m7m6bma3n6hnk15wxw5fd7sn9y7x-develkit/src -I /nix/store/w218m7m6bma3n6hnk15wxw5fd7sn9y7x-develkit/objs -I objs/addon/ndk -I /usr/include/libxml2 -I src/stream \
       >   -o objs/addon/xjls3h70j5zj2pipyvlhl1zwd4c14wf8-vod/ngx_buffer_cache.o \
       >        /nix/store/xjls3h70j5zj2pipyvlhl1zwd4c14wf8-vod/ngx_buffer_cache.c
       > cc1: all warnings being treated as errors
       > make[1]: *** [objs/Makefile:2649: objs/addon/filters/audio_decoder.o] Error 1
       > make[1]: *** Waiting for unfinished jobs....
       > make[1]: *** [objs/Makefile:2656: objs/addon/filters/audio_encoder.o] Error 1
       > make[1]: *** [objs/Makefile:2670: objs/addon/thumb/thumb_grabber.o] Error 1
       > make[1]: Leaving directory '/build/nginx-1.26.2'
       > make: *** [Makefile:10: build] Error 2
```

### Additional context

Most likely from an `nginx` module. And likely this one: https://github.com/NixOS/nixpkgs/blob/4734b2c5fe9aeb2eb09fa707614e8142b27d72a4/nixos/modules/services/video/frigate.nix#L172

### Notify maintainers

<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

@mweinelt 

### Metadata

<!-- Please insert the output of running `nix-shell -p nix-info --run "nix-info -m"` below this line -->
 - system: `"x86_64-linux"`
 - host os: `Linux 6.11.2, NixOS, 24.11 (Vicuna), 24.11.20241009.5633bcf`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.8`
 - channels(root): `"nixos"`
 - channels(bill): `"home-manager"`
 - nixpkgs: `/nix/store/60sn02zhawl3kwn0r515zff3h6hg6ydz-source`

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/servers/http/nginx/modules.nix
+++ b/pkgs/servers/http/nginx/modules.nix
@@ -11,6 +11,7 @@
 , expat
 , fdk_aac
 , ffmpeg-headless
+, ffmpeg_6-headless
 , geoip
 , libbsd
 , libiconv
@@ -991,7 +992,7 @@ let self = {
       '';
     };
 
-    inputs = [ ffmpeg-headless fdk_aac openssl libxml2 libiconv ];
+    inputs = [ ffmpeg_6-headless fdk_aac openssl libxml2 libiconv ];
 
     meta = with lib; {
       description = "VOD packager";

